streaming/stream_session.cc

```diff
@@ -45,10 +45,41 @@ static sstables::offstrategy is_offstrategy_supported(streaming::stream_reason r
     static const std::unordered_set<streaming::stream_reason> operations_supported = {
         streaming::stream_reason::bootstrap,
         streaming::stream_reason::replace,
+        streaming::stream_reason::removenode,
+        streaming::stream_reason::decommission,
+        streaming::stream_reason::repair,
+        streaming::stream_reason::rebuild,
     };
```

Comment:
for rebuild we can probably have explicit trigger as we do for bootstrap and replace, but that can follow.

Reply:
Yes, we can have explicit trigger for some of ops. Since we are moving more and more to automatic trigger, it is not very important though.